main.py

```diff
@@ -312,7 +308,8 @@ def update(self, circuit_id):
         status = 200
 
         log.debug("update result %s %s", result, status)
-        emit_event(self.controller, "updated", evc_id=evc.id, data=data)
```

Comment:
Here for the event `kytos/mef_eline.updated`, the content included the data being modifies. I replaced this with `{"id", "name", "metadata", "active", "enabled", "uni_a", "uni_z"}`. Do I add `data` to the content too? In case something was modified and it is not inside the current content like `map_evc_event_content(evc, data)`.

Reply:
@Alopalao, yes, let's also merge the `data` into the dict as you've done maintaining the semantics for the other events that had extra keys.

Reply:
Done on commit [faec31c5eba8adc4cc9fd0308f54b694c14cab95](https://github.com/kytos-ng/mef_eline/pull/257/commits/faec31c5eba8adc4cc9fd0308f54b694c14cab95)